backend/LexBoxApi/Services/EmailService.cs

```diff
@@ -206,13 +209,14 @@ private record RenderResult(string Subject, string Html);
         message.Body = new TextPart(TextFormat.Html) { Text = renderResult.Html };
     }
 
-    private static MimeMessage StartUserEmail(User user, string? email = null)
+    private static MimeMessage? StartUserEmail(User user, string? email = null)
     {
         return StartUserEmail(user.Name, email ?? user.Email);
     }
 
-    private static MimeMessage StartUserEmail(string name, string email)
+    private static MimeMessage? StartUserEmail(string name, string? email)
     {
+        if (string.IsNullOrEmpty(email)) return null;
```

Comment:
I'd rather remove this from here, and do this in the overload that takes the user. Then change the email parameter back to being non nullable. The reason is because now other places that call this with a non null email have to handle the null return case.

Reply:
Done in commit 96f0acee.